content/documentation/pages/3-stream-developer-guides/2-streams/1-standalone-stream-sample.md

```diff
@@ -481,29 +468,29 @@ In this step, we create the `UsageCostLogger` sink.
 
 <!--Kafka-->
 
-You can either [directly download the project generated by the Spring Initialzr](https://start.spring.io/#!type=maven-project&language=java&platformVersion=2.4.3.RELEASE&packaging=jar&jvmVersion=1.8&groupId=io.spring.dataflow.sample&artifactId=usage-cost-logger-kafka&name=usage-cost-logger-kafka&description=Demo%20project%20for%20Spring%20Boot&packageName=io.spring.dataflow.sample.usagecostlogger&dependencies=cloud-stream,actuator,kafka)
+You can either [directly download the project generated by the Spring Initialzr](https://start.spring.io/#!type=maven-project&language=java&platformVersion=2.4.3.RELEASE&packaging=jar&jvmVersion=1.8&groupId=io.spring.dataflow.sample&artifactId=usage-cost-logger-kafka&name=usage-cost-logger-kafka&description=Demo%20project%20for%20Spring%20Boot&packageName=io.spring.dataflow.sample.usagecostlogger&dependencies=cloud-stream,actuator,kafka) and click on **Generate Project**.
```

Comment:
If you add `starter.zip?` to the URL it will download automatically for you, and you won't have to be taken to the page.   For example:
`https://start.spring.io/starter.zip?type=maven-project&language=java&platformVersion=2.4.3.RELEASE&packaging=jar&jvmVersion=1.8&groupId=io.spring.dataflow.sample&artifactId=usage-detail-sender-kafka&name=usage-detail-sender-kafka&description=Demo%20project%20for%20Spring%20Boot&packageName=io.spring.dataflow.sample.usagedetailsender&dependencies=cloud-stream,actuator,kafka`